src/Routes/RosSystemDetail/ros-details-page.scss

```diff
@@ -17,3 +17,12 @@
   position: relative;
   padding-bottom: 72px;
 }
+.rosDetailsHead {
+  .ins-entity-detail {
+    .ins-entity-facts {
+      .pf-l-flex:last-child {
+        display: none !important;
+      }
```

Comment:
@r14chandra @kgaikwad  Normally it is recommended to avoid using important as much as possible as it overrides all previous styling rules for that specific property but since this is what we want to do here to hide the element as per me it is okay to use this 👍🏻

Reply:
Thanks @PreetiW for reviewing!